lib/hammer_cli_foreman/hosts/common_update_options.rb

```diff
@@ -77,6 +77,20 @@ def request_params
         if action == :update
           params['host']['compute_attributes']['volumes_attributes'] = nested_attributes(option_volume_list) unless option_volume_list.empty?
           params['host']['interfaces_attributes'] = interfaces_attributes unless option_interface_list.empty?
+          new_location = option_new_location_name || option_new_location_title
```

Comment:
Could this logic be extracted to the OptionSource so that it is resolved along with other options and reusable like the new OptionBuilder